packages/react-components/src/components/data-providers/tsp-data-provider.ts

```diff
@@ -39,11 +41,11 @@ export class TspDataProvider {
             };
         }
 
-        this.totalRange = this.timeGraphEntries[0].end - this.timeGraphEntries[0].start; // 1332170682540133097 - starttime
+        this.totalRange = totalTimeRange.getEnd() - totalTimeRange.getstart();
         let fetchParameters = QueryHelper.selectionTimeQuery(QueryHelper.splitRangeIntoEqualParts(1332170682440133097, 1332170682540133097, 1120), ids);
         if (viewRange && resolution) {
-            const start = viewRange.start + this.timeGraphEntries[0].start;
-            const end = viewRange.end + this.timeGraphEntries[0].start;
+            const start = viewRange.start + totalTimeRange.getstart();
```

Comment:
I would suggest to swap the order in next two lines (totalTimeRange.getstart() + viewRange.start/end)

Reply:
Will do